haystack/agents/base.py

```diff
@@ -165,15 +176,15 @@ def run(
                 return self._format_answer(query=query, transcript=transcript, answer=final_answer)
             tool_name, tool_input = self._extract_tool_name_and_tool_input(pred=preds[0])
             if tool_name is None or tool_input is None:
-                raise AgentError(f"Wrong output format. Transcript:\n{transcript}")
+                raise AgentError(f"Wrong output format. Transcript:\n{transcript}") #is there any way to fix it?
```

Comment:
Malte also suggested a change for this line in this open PR: https://github.com/deepset-ai/haystack/pull/4269/files